packages/emulation/libretro-mrboom/package.mk

```diff
@@ -2,11 +2,12 @@
 # Copyright (C) 2016-present Team LibreELEC (https://libreelec.tv)
 
 PKG_NAME="libretro-mrboom"
-PKG_VERSION="e074bafa1fe9480df1ba412752912a2c2c454958"
-PKG_SHA256="ac337b339cb1da9d013745c72b59b84ee64d94ada8f303be671e680d50061c15"
-PKG_LICENSE="GPLv3"
-PKG_SITE="https://github.com/libretro/mrboom-libretro"
-PKG_URL="https://github.com/libretro/mrboom-libretro/archive/${PKG_VERSION}.tar.gz"
+PKG_VERSION="5.2"
+PKG_SUBVER=".454d614"
+PKG_SHA256="50e4fe4bc74b23ac441499c756c4575dfe9faab9e787a3ab942a856ac63cf10d"
+PKG_LICENSE="MIT"
+PKG_SITE="https://github.com/Javanaise/mrboom-libretro"
+PKG_URL="https://github.com/Javanaise/mrboom-libretro/releases/download/${PKG_VERSION}/MrBoom-src-${PKG_VERSION}${PKG_SUBVER}.tar.gz"
```

Comment:
```suggestion
PKG_URL="https://github.com/Javanaise/mrboom-libretro/releases/download/${PKG_VERSION}/MrBoom-src-${PKG_VERSION}.tar.gz"
```

Reply:
this will make PKG_URL 404.

Reply:
I have a look

Reply:
It's due to the first reference to `${PKG_VERSION}` after `/download/` now also containing the subversion.

Reply:
ah damn its ugly the one or the other way ;D
we can clean up at the next bump

Reply:
`PKG_URL="https://github.com/Javanaise/mrboom-libretro/releases/download/$(get_pkg_version_maj_min)/MrBoom-src-${PKG_VERSION}.tar.gz"`

Reply:
the next version is likely 5.3 again so we can go back to a normal versioning